coordinator/client/zookeeper/connection_test.go

```diff
@@ -23,7 +23,7 @@ import (
 	"time"
```

Comment:
We should probably add a test like Create("/a/b/c/d/e") to validate ensurePath().